.repair-bot/repro.py

```
@@ -0,0 +1,98 @@
+
+def main():
+    failures = []
+
+    for name, check_fn in [
+        ("upstream_exports", check_upstream_exports),
+        ("custom_pipeline_cleanup", check_custom_pipeline_cleanup),
+        ("dense_gemm_imports", check_dense_gemm_imports),
+    ]:
+        try:
+            check_fn()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)